react website v1!

chschlich opened the pull request:
made a newer and slightly better looking homepage in react

## Description
Updated the homepage using React with Typescript to slightly improve it visually. It's still a little visually unappealing but hopefully it's closer to what the final product will look like.

## Closed Issues
Does not close any issues.

chschlich committed:
react website v1!

made a newer and slightly better looking homepage in react

codeJester27 commented:
plz rename the folder from my-vue-app to something else, vue is a completely different framework than react so its a little confusing

chschlich committed:
my-vue-app rename

rename

chschlich commented:
of course, glory to boardcorp

codeJester27 commented on WebApp/app.go:
```diff
@@ -323,8 +323,10 @@ func main() {
-	logged.StaticFile("/", "./website/homepage.html")
-	logged.Static("/static", "./website")
+	logged.StaticFile("/", "./homepage/boardsite/dist/index.html")
+	logged.StaticFile("/image2vector (2).svg", "./homepage/boardsite/dist/image2vector (2).svg")
```
might want to change that file name

codeJester27 commented on WebApp/app.go:
```diff
@@ -323,8 +323,10 @@ func main() {
-	logged.Static("/static", "./website")
+	logged.StaticFile("/", "./homepage/boardsite/dist/index.html")
+	logged.StaticFile("/image2vector (2).svg", "./homepage/boardsite/dist/image2vector (2).svg")
+	logged.StaticFile("/vite.svg", "./homepage/boardsite/dist/vite.svg")
```
assuming we are not leaving the vite logo on our page at all, you can just remove this file and stop serving it

codeJester27 commented on WebApp/homepage/boardsite/src/routes/assetsReal/react.svg:
unused

codeJester27 commented on WebApp/homepage/boardsite/src/routes/root.tsx:
```diff
@@ -0,0 +1,36 @@
+    <>
+      <img src={boardmeshLogo} className="logoMain" alt="Boardwalk logo" />
+      <h1>
+        Boardmesh: The API that translates whiteboard drawings into regions!
```
We'll have to look into adjusting the wording here

codeJester27 reviewed (commented):
General note is that the file structure is a little weird for the image assets which we will fix before merging

chschlich committed:
push for check

localhost:8080 is pulling up a dist folder that no longer exists, is it pulling from memory?

codeJester27 reviewed (changes requested):
hold

jaredtalbot committed:
nit: change things

codeJester27 reviewed (approved):
no guarantees that the first deploy works but I tried™️
